shortcuts.sh

```diff
@@ -132,19 +129,19 @@ function swgjs_create_amp_release_pr() {
 
   __swgjs_get_github_username
 
-  SWG_VERSION=$1
-  if [[ -z "$SWG_VERSION" ]]; then
-    echo "What SwG Version are you releasing? (ex: 0.1.22.333)"
-    read SWG_VERSION
-  fi
-
   swgjs_install
   __swgjs_install_amp
 
-  # Build Swgjs for AMP.
-  cd $SWGJS_PATH
+  # Switch to clean AMP branch.
   git fetch team
-  git checkout team/main
+  git checkout team/amp
+  if [[ `git status --porcelain` ]]; then
+    # Avoid building with local changes.
+    return
+  fi
+
+  # Build Swgjs for AMP.
+  SWG_VERSION=$(git rev-parse --short HEAD)
```

Comment:
I think that pushing from HEAD should generally be avoided because it could point to an unexpected revision when the script runs (for example, if one engineer pushes a commit while another engineer is making the release). That might not be an issue here if we aren't expecting much activity on the amp branch though.

Reply:
Yeah, this seems safe for the `amp` branch specifically, since the same people working on the branch will be kicking of the release on-demand